### PR TITLE
lsr-role2collection.py - improves collection mapping

### PR DESCRIPTION
**Pull request by nhosoi**
- bug fix: README needs to be copied to the doc dir after the extra-mapping..
- improve --extra-mapping to support mapping fedora.linux_system_roles to redhat.rhel_system_roles.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,6 +195,13 @@ lsr_role2collection.py [-h] [--namespace NAMESPACE] [--collection COLLECTION]
                      in the README that you do not want to change, you may have to change the
                      README in another way, not using this script, by using sed with a custom
                      regex.
+--collection-mapping If it is set and given namespace.collection is not default, the default
+                     collection fedora.linux_system_roles is converted to
+                     NAMESPACE.COLLECTION.
```
Comment (richm) [inline README.md:200]: I thought we could already do this with `--extra-mapping`?

File: README.md [hunk 2]
```diff
@@ -194,6 +195,13 @@ lsr_role2collection.py [-h] [--namespace NAMESPACE] [--collection COLLECTION]
...
+--meta-runtime META_RUNTIME
+                     This is the path to the collection meta/runtime.yml - the default is
+                     /home/nhosoi/linux-system-roles/auto-
+                     maintenance/lsr_role2collection/runtime.yml.
```
Comment (richm) [inline README.md:204]: This doesn't look right

File: README.md (hunk 1 above)
Comment (nhosoi) [reply README.md:200]: No, `--extra-mapping` unfortunately does not cover this case for now...

Maybe, I'd better improve `parse_extra_mapping` to handle it, instead...

File: README.md (hunk 2 above)
Comment (nhosoi) [reply README.md:204]: ??? I just copied the Usage since it was missing... If it's not a good idea, I'll drop it. :)

Comment (richm) [reply README.md:204]: I mean you have `/home/nhosoi` there - should it be `$HOME/linux-system-roles/auto-maintenance/lsr_role2collection/runtime.yml` instead?

File: README.md (hunk 1 above)
Comment (richm) [reply README.md:200]: ok - if `--extra-mapping` doesn't work, then add `--collection-mapping`

Comment (nhosoi) [reply README.md]: Thanks, @richm. I've modified the parser a bit to support it.

File: README.md (hunk 2 above)
Comment (nhosoi) [reply README.md]: Ahhh. Sorry, I was blind... :p Then, I'm adding them back with $HOME...

File: lsr_role2collection.py [hunk 3]
```diff
@@ -1088,7 +1088,9 @@ def role2collection():
             "This is a comma delimited list of extra mappings to apply when "
             "converting the files - this replaces the given role name with "
             "collection format with the optional given namespace and collection "
-            "as sell as the given FQCN with other FQCN."
+            "as sell as the given FQCN with other FQCN. In addition, 'fedora."
```
Comment (richm) [inline lsr_role2collection.py:1091]: as well as

Comment (nhosoi) [reply lsr_role2collection.py]: *shame shame* Thanks!

**Review by richm — APPROVED** [review]

**Comment by nhosoi** [comment]
Sorry, @richm. I found a bug in this patch. :( Debugging it now...

**Pull request merged**